controllers/plantController.js

```diff
@@ -0,0 +1,18 @@
+const { catchAsync } = require("../utils/error");
+
+const plantService = require("../services/plantService");
+
+const plantsDetails = catchAsync(async (req, res) => {
+  const { plantId } = req.params;
+  if (!plantId) {
+    throw new Error("KEY_ERROR");
+  }
+
+  const plantsDetails = await plantService.plantsDetails(plantId);
+
+  res.status(201).json({ message: plantsDetails });
```

Comment:
```suggestion
  return res.status(200).json({ message: plantsDetails });
```
명확한 status code를 return 하도록 수정해주세요~